x-pack/plugins/observability/public/components/shared/exploratory_view/configurations/constants/labels.ts

```diff
@@ -314,17 +314,38 @@ export const MOBILE_APP = i18n.translate('xpack.observability.expView.fieldLabel
   defaultMessage: 'Mobile App',
 });
 
-export const MEMORY_USAGE = i18n.translate(
+export const SYSTEM_MEMORY_USAGE = i18n.translate(
   'xpack.observability.expView.fieldLabels.mobile.memoryUsage',
   {
-    defaultMessage: 'Memory Usage',
+    defaultMessage: 'System memory usage',
```

Comment:
Nice fix of capitalization to comply with [EUI writing guidelines](https://elastic.github.io/eui/#/guidelines/writing/guidelines#capitalization)